java/client/src/main/java/glide/api/commands/PubSubClusterCommands.java

```diff
@@ -46,4 +47,70 @@ public interface PubSubClusterCommands {
      * }</pre>
      */
     CompletableFuture<String> publish(GlideString message, GlideString channel, boolean sharded);
+
+    /**
+     * Lists the currently active shard channels.
+     *
+     * @apiNote When in cluster mode, the command is routed to all nodes, and aggregates the response
+     *     into a single array.
+     * @see <a href="https://valkey.io/commands/pubsub-shardchannels/">valkey.io</a> for details.
+     * @return An <code>array</code> of currently active shard channels matching the given pattern. If
+     *     no pattern is specified, all active shard channels are returned.
```

Comment:
```suggestion
     * @return An <code>array</code> of all active shard channels.
```
There is no pattern in this API